spec/models/profile_spec.rb

```diff
@@ -29,4 +29,14 @@
 	    expect(profile).to_not be_valid
 	  end
   end
+
+  it "returns correct age" do
+    profile = FactoryGirl.build(:profile)
+    expect(profile.age).to eq(15)
+  end
+
+  it "returns full name" do
+    profile = FactoryGirl.build(:profile)
+    expect(profile.name).to eq("Karl Doe")
```

Comment:
What happens when the factory is changed and creates different first and last names?
What about `expect(profile.name).to eq("#{profile.first_name} #{profile.last_name})`